doc/variableexplorer.rst

```diff
@@ -2,95 +2,213 @@
 Variable Explorer
 #################
 
-The **Variable Explorer** shows the namespace contents (all global object references, such as variables, functions, modules, etc.) of the currently selected :doc:`ipythonconsole` session, and allows you to interact with them through a variety of GUI-based editors.
+The **Variable Explorer** allows you to view and manage interactively the variables previously generated in the Editor or the IPython Console. 
 
-.. image:: images/variable_explorer/variable_explorer_standard.png
+.. image:: images/variable_explorer/variable-explorer-standard.png
    :align: center
    :alt: Spyder Variable Explorer, with a list of variables and their contents
 
 |
 
+It shows the namespace contents (all global object references, such as variables, functions, modules, etc.) of the currently selected :doc:`ipythonconsole` session, and allows you to modify their value, add more values or delete them through a variety of GUI-based editors.
 
-====================
-Features and editors
-====================
+.. image:: images/variable_explorer/variable-explorer-execution.gif
+   :align: center
+   :alt: Spyder Variable Explorer execution with a variable of type list
+
+|
+
+The Variable Explorer gives you information on the name, size, type and value of each one of the variables. To modify the value of one of them, you can double click it in the pane and write its new value.
+
+.. image:: images/variable_explorer/variable-explorer-modifying.gif
+   :align: center
+   :alt: Spyder Variable Explorer modifying value of a variable 
+
+|
+
+==============
+Object Viewers
+==============
+
+Spyder's :guilabel:`Variable Explorer` offers built in support for editing lists, strings, dictionaries, NumPy arrays, Pandas DataFrames, and more, and can also histogram, plot, or even display some of them as an RGB image. Several examples of this functionality are shown below:
 
-Spyder's :guilabel:`Variable Explorer` offers built in support for editing lists, strings, dictionaries, NumPy arrays, Pandas DataFrames, and more, and can also histogram, plot, or even display some of them as an RGB image.
-Several examples of this functionality follow:
 
 |
 
-.. image:: images/variable_explorer/variable_explorer_text_long.png
+~~~~~~~
+Strings
+~~~~~~~
+
+When a string variable is longer than forty characters, you can double click it to see its value in a text editor where you can modify it.
+
+.. image:: images/variable_explorer/variable-explorer-text-long.png
    :align: center
    :alt: Variable Explorer text editor, displaying a long string in a window
 
 |
 
-.. image:: images/dialog/dialog_user_env_variables_edit.png
+~~~~~~~~~~~~
+Dictionaries
+~~~~~~~~~~~~
+
+When the type of a variable is a dictionary, double clicking it will show you a viewer displaying each one of the keys of the dictionary along with its associated value. You can double click any of the values to modify it. 
```

Comment:
```suggestion
When the type of a variable is a dictionary, double clicking it will show a viewer displaying each one of the keys of the dictionary along with its associated value. You can double click any of the values to modify it. 
```